superset/assets/src/dashboard/components/SliceHeader.jsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import moment from 'moment';
+import { connect } from 'react-redux';
 
 import { t } from '../../locales';
 import EditableTitle from '../../components/EditableTitle';
 import TooltipWrapper from '../../components/TooltipWrapper';
 
 const propTypes = {
   slice: PropTypes.object.isRequired,
+  superset_can_explore: PropTypes.bool,
```

Comment:
Let's use camel case for props. The guideline is it's ok to reference objects keys coming out of python in underscore notation, but for everything declared on the js side (const, let, props) we use camel case.

Reply:
Sounds good! Will do. Yes, there are other places where snake case is used in the js besides the examples I gave -- could use a pass to fix them all up for sure.